[2021-12-15 04:58:12,502] {processor.py:163} INFO - Started process (PID=64123) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 04:58:12,503] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 04:58:12,503] {logging_mixin.py:109} INFO - [2021-12-15 04:58:12,503] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 04:58:12,530] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 04:58:12,535] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:12,546] {logging_mixin.py:109} INFO - [2021-12-15 04:58:12,546] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:58:12,556] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:12,560] {logging_mixin.py:109} INFO - [2021-12-15 04:58:12,560] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 04:58:12,571] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.072 seconds
[2021-12-15 05:04:43,128] {processor.py:163} INFO - Started process (PID=1006) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:04:43,129] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:04:43,129] {logging_mixin.py:109} INFO - [2021-12-15 05:04:43,129] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:04:43,157] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:04:43,162] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:43,174] {logging_mixin.py:109} INFO - [2021-12-15 05:04:43,174] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:04:43,184] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:43,188] {logging_mixin.py:109} INFO - [2021-12-15 05:04:43,188] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:04:43,198] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:05:17,876] {processor.py:163} INFO - Started process (PID=1221) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:17,877] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:05:17,877] {logging_mixin.py:109} INFO - [2021-12-15 05:05:17,877] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:17,906] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:17,911] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:17,923] {logging_mixin.py:109} INFO - [2021-12-15 05:05:17,922] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:05:17,933] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:17,938] {logging_mixin.py:109} INFO - [2021-12-15 05:05:17,938] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:05:17,950] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.078 seconds
[2021-12-15 05:05:52,729] {processor.py:163} INFO - Started process (PID=1426) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:52,730] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:05:52,730] {logging_mixin.py:109} INFO - [2021-12-15 05:05:52,730] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:52,757] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:05:52,762] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:52,773] {logging_mixin.py:109} INFO - [2021-12-15 05:05:52,773] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:05:52,782] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:52,787] {logging_mixin.py:109} INFO - [2021-12-15 05:05:52,787] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:05:52,796] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.070 seconds
[2021-12-15 05:06:26,446] {processor.py:163} INFO - Started process (PID=1544) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:06:26,447] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:06:26,448] {logging_mixin.py:109} INFO - [2021-12-15 05:06:26,448] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:06:26,483] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:06:26,487] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:26,530] {logging_mixin.py:109} INFO - [2021-12-15 05:06:26,530] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:06:26,541] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:26,546] {logging_mixin.py:109} INFO - [2021-12-15 05:06:26,546] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:06:26,557] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.113 seconds
[2021-12-15 05:07:05,350] {processor.py:163} INFO - Started process (PID=1690) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:05,351] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:07:05,352] {logging_mixin.py:109} INFO - [2021-12-15 05:07:05,352] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:05,380] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:05,384] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:05,396] {logging_mixin.py:109} INFO - [2021-12-15 05:07:05,396] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:05,406] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:05,410] {logging_mixin.py:109} INFO - [2021-12-15 05:07:05,410] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:07:05,421] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:07:40,199] {processor.py:163} INFO - Started process (PID=1775) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:40,200] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:07:40,201] {logging_mixin.py:109} INFO - [2021-12-15 05:07:40,201] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:40,228] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:07:40,233] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:40,245] {logging_mixin.py:109} INFO - [2021-12-15 05:07:40,245] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:40,255] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:40,259] {logging_mixin.py:109} INFO - [2021-12-15 05:07:40,259] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:07:40,270] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:08:14,516] {processor.py:163} INFO - Started process (PID=1856) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:14,517] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:08:14,517] {logging_mixin.py:109} INFO - [2021-12-15 05:08:14,517] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:14,545] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:14,549] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:14,561] {logging_mixin.py:109} INFO - [2021-12-15 05:08:14,561] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:08:14,571] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:14,575] {logging_mixin.py:109} INFO - [2021-12-15 05:08:14,575] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:08:14,586] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.073 seconds
[2021-12-15 05:08:51,762] {processor.py:163} INFO - Started process (PID=1972) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:51,764] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:08:51,764] {logging_mixin.py:109} INFO - [2021-12-15 05:08:51,764] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:51,791] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:08:51,796] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:51,808] {logging_mixin.py:109} INFO - [2021-12-15 05:08:51,808] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:08:51,818] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:51,822] {logging_mixin.py:109} INFO - [2021-12-15 05:08:51,822] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:08:51,834] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:09:31,261] {processor.py:163} INFO - Started process (PID=2175) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:09:31,262] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:09:31,263] {logging_mixin.py:109} INFO - [2021-12-15 05:09:31,262] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:09:31,291] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:09:31,295] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:31,307] {logging_mixin.py:109} INFO - [2021-12-15 05:09:31,307] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:09:31,317] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:31,322] {logging_mixin.py:109} INFO - [2021-12-15 05:09:31,322] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:09:31,332] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:10:07,268] {processor.py:163} INFO - Started process (PID=2388) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:07,269] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:10:07,270] {logging_mixin.py:109} INFO - [2021-12-15 05:10:07,270] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:07,295] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:07,300] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:07,311] {logging_mixin.py:109} INFO - [2021-12-15 05:10:07,311] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:07,322] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:07,326] {logging_mixin.py:109} INFO - [2021-12-15 05:10:07,326] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:10:07,337] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.071 seconds
[2021-12-15 05:10:48,042] {processor.py:163} INFO - Started process (PID=2648) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:48,043] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:10:48,044] {logging_mixin.py:109} INFO - [2021-12-15 05:10:48,043] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:48,070] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:10:48,074] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:48,086] {logging_mixin.py:109} INFO - [2021-12-15 05:10:48,086] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:48,096] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:48,100] {logging_mixin.py:109} INFO - [2021-12-15 05:10:48,100] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:10:48,111] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.072 seconds
[2021-12-15 05:11:27,746] {processor.py:163} INFO - Started process (PID=2916) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:11:27,747] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:11:27,747] {logging_mixin.py:109} INFO - [2021-12-15 05:11:27,747] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:11:27,774] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:11:27,778] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:27,790] {logging_mixin.py:109} INFO - [2021-12-15 05:11:27,790] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:11:27,800] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:27,804] {logging_mixin.py:109} INFO - [2021-12-15 05:11:27,804] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:11:27,816] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.072 seconds
[2021-12-15 05:12:04,171] {processor.py:163} INFO - Started process (PID=3025) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:04,173] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:12:04,173] {logging_mixin.py:109} INFO - [2021-12-15 05:12:04,173] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:04,201] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:04,205] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:04,218] {logging_mixin.py:109} INFO - [2021-12-15 05:12:04,218] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:12:04,227] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:04,232] {logging_mixin.py:109} INFO - [2021-12-15 05:12:04,232] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:12:04,242] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.074 seconds
[2021-12-15 05:12:40,818] {processor.py:163} INFO - Started process (PID=3195) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:40,819] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:12:40,820] {logging_mixin.py:109} INFO - [2021-12-15 05:12:40,819] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:40,846] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:12:40,852] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:40,864] {logging_mixin.py:109} INFO - [2021-12-15 05:12:40,864] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:12:40,874] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:40,879] {logging_mixin.py:109} INFO - [2021-12-15 05:12:40,879] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:12:40,891] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.076 seconds
[2021-12-15 05:13:18,293] {processor.py:163} INFO - Started process (PID=3323) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:18,294] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:13:18,294] {logging_mixin.py:109} INFO - [2021-12-15 05:13:18,294] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:18,321] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:18,326] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:18,337] {logging_mixin.py:109} INFO - [2021-12-15 05:13:18,337] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:18,347] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:18,351] {logging_mixin.py:109} INFO - [2021-12-15 05:13:18,351] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:13:18,361] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.071 seconds
[2021-12-15 05:13:57,530] {processor.py:163} INFO - Started process (PID=3480) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:57,531] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:13:57,532] {logging_mixin.py:109} INFO - [2021-12-15 05:13:57,531] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:57,559] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:13:57,563] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:57,575] {logging_mixin.py:109} INFO - [2021-12-15 05:13:57,575] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:57,586] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:57,591] {logging_mixin.py:109} INFO - [2021-12-15 05:13:57,591] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:13:57,602] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.075 seconds
[2021-12-15 05:14:35,277] {processor.py:163} INFO - Started process (PID=3702) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:14:35,278] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2021-12-15 05:14:35,279] {logging_mixin.py:109} INFO - [2021-12-15 05:14:35,279] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:14:35,306] {processor.py:632} INFO - DAG(s) dict_keys(['example_complex']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py
[2021-12-15 05:14:35,311] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:35,323] {logging_mixin.py:109} INFO - [2021-12-15 05:14:35,323] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:14:35,333] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:35,337] {logging_mixin.py:109} INFO - [2021-12-15 05:14:35,337] {dag.py:2914} INFO - Setting next_dagrun for example_complex to None
[2021-12-15 05:14:35,349] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/example_complex.py took 0.075 seconds
